The simulator panics when we invoke the `help`

Invoking `bin/simulator --help` successfully prints the help screen but it panics at the end. We should probably investigate why.

```sh
[sub]Command required
usage: simulator <Command> [-h|--help] [--cleanup] [--log-level "<value>"]
                 [--enable-stdout-logs]

                 HyperSDK program VM simulator

Commands:

  run             Run a HyperSDK program simulation plan
  program-create  Create a HyperSDK program transaction
  key-create      Creates a new named private key and stores it in the database
  interpreter     Read input from a buffered stdin

Arguments:

  -h  --help                Print help information
      --cleanup             remove simulator directory on exit. Default: true
      --log-level           log level. Default: info
      --enable-stdout-logs  enable displaying logs in stdout. Default: false


panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x28 pc=0x10303c208]

goroutine 1 [running]:
github.com/ava-labs/hypersdk/x/programs/cmd/simulator/cmd.(*Simulator).Execute(0x14000118500, {0x104251af0, 0x14000118190})
	/Users/hello/hypersdk/x/programs/cmd/simulator/cmd/root.go:143 +0x198
main.runSimulator()
	/Users/hello/hypersdk/x/programs/cmd/simulator/simulator.go:28 +0x64
main.main()
	/Users/hello/hypersdk/x/programs/cmd/simulator/simulator.go:15 +0x1c
```

--- a/x/programs/cmd/simulator/cmd/root.go
+++ b/x/programs/cmd/simulator/cmd/root.go
@@ -140,7 +140,6 @@ func (s *Simulator) Execute(ctx context.Context) error {
 
 	err := s.ParseCommandArgs(ctx, os.Args, false)
 	if err != nil {
-		s.log.Error("error when parsing command args", zap.Error(err))
 		return err
 	}
 

